src/frontend/screens/Observation/Buttons.tsx

```diff
@@ -45,6 +75,14 @@ export const ButtonFields = ({
   const {formatMessage: t} = useIntl();
   const navigation = useNavigationFromRoot();
   const deleteObservationMutation = useDeleteObservation();
+  const {observation, preset} = useObservationWithPreset(observationId);
+  const attachmentUrlQueries = useAttachmentUrlQueries(
+    observation.attachments,
+    'original',
+  );
+  const attachmentBase64Queries = useAttachmentsBase64Query(
+    attachmentUrlQueries.filter(res => !!res.data).map(res => res.data!),
```

Comment:
See comment above about having 2 queries, BUT if it is necessary to have both queries, this second query should be a [dependent query](https://tanstack.com/query/latest/docs/framework/react/guides/dependent-queries).

Reply:
It's hard to make dependant queries while depending on `useQueries`. But I made slight adjustments so there's not that much filtering going on here. Let me know if that's little better.